test/test_order.py

```diff
@@ -183,8 +183,112 @@ def test_order_cancel(self):
         self.assertEqual(po.status, 40)
         self.assertEqual(po.status_text, "Cancelled")
 
+    def test_order_complete_with_receive(self):
+        """Test that we can complete an order via the API, after receiving
+        items via API"""
+
+        n = len(order.PurchaseOrder.list(self.api)) + 1
+        ref = f"PO-{n}"
+
+        # First, let's create a new PurchaseOrder
+        po = order.PurchaseOrder.create(self.api, data={
+            'supplier': 1,
+            'reference': ref,
+            'description': 'A purchase order with items to be received',
```

Comment:
Question: Should we translate this?

Reply:
What exactly? I can't see that test data is translated in general, and I think that's a bit overkill..

Reply:
My error, sorry @miggland. I am so routine-blind regarding untranslated strings that I did not consider that it is in a test file.